src/ol_infrastructure/components/aws/cache.py

```diff
@@ -234,7 +236,9 @@ def redis(
             automatic_failover_enabled=True,
             cluster_mode=cluster_mode_param,
             engine="redis",
-            engine_version=cache_config.engine_version,
+            engine_version="6.x"
```

Comment:
Why did we set this here?

Reply:
That's a conditional because it's expecting the string `6.x` for the version if we are running anything >= v6